docs/js/custom.js

```diff
@@ -44,6 +44,7 @@ document.addEventListener("DOMContentLoaded", function () {
   script.setAttribute("version", "stable");
   script.setAttribute("runllm-keyboard-shortcut", "Mod+j"); // cmd-j or ctrl-j to open the widget.
   script.setAttribute("runllm-name", "Daft");
+  script.setAttribute("runllm-position", "BOTTOM_LEFT");
   script.setAttribute("runllm-position-x", "20px");
   script.setAttribute("runllm-position-y", "98px");
```

Comment:
```suggestion
  script.setAttribute("runllm-position-y", "20px");
```

It looks like we had all this extra padding to move the "Ask AI" above the readthedocs versions. Now that it's on the left, I personally like the x/y matching